src/strawberry_sqlalchemy_mapper/scalars.py

```diff
@@ -0,0 +1,10 @@
+from typing import Union
+
+import strawberry
+
+Int64 = strawberry.scalar(
```

Comment:
Oh, I would much rather pass it as an integer rather than a string, honestly! I was just following the example from:
https://strawberry.rocks/docs/types/scalars#int64
Do you think there's a better way in order to preserve it as an integer?

Reply:
So, you can set the GRAPHQL_MAX_INT, which from our limited testing "works", but the docs recommend doing what you did here creating a new scalar serialized as a string, primarily because javascript only supports up to 53 bits. 

I like what you have here, just slightly prefer the name BigInt to Int64, and possibly also worth including a warning docstring to be careful with these strings in javascript clients.

Reply:
I renamed it, do let me know if there's anything else before it can be merged!